database/migrations/2021_02_25_175000_create_booking_categories_table.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateBookingCategoriesTable extends Migration
+{
+    public function up()
+    {
+        Schema::create('booking_categories', function (Blueprint $table) {
+            $table->id();
+            $table->string('label');
+            $table->timestamps();
+        });
+    }
+}
```

Comment:
Does this need to have its own database table?

Could we do a similar implementation to Keyword Types (https://github.com/tipoff/seo/pull/28)?

Reply:
The reason models are used is a possibility to extend resource with configuration in the future. Booking category can define config for a group of bookings, for example category can have "requires_waiver" where one bookings need it, other ones dont.

Reply:
OK, that makes sense. Thanks

Reply:
I created issue #40 for this